Traceback (most recent call last):
  File "paths_constraints_main.py", line 5, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/__init__.py", line 53, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/project.py", line 738, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/factory.py", line 325, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/sim_manager.py", line 831, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/exploration_techniques/__init__.py", line 173, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/exploration_techniques/loop_seer.py", line 4, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/knowledge_base/__init__.py", line 2, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/knowledge_base/knowledge_base.py", line 16, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/knowledge_plugins/__init__.py", line 3, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/knowledge_plugins/variables/__init__.py", line 2, in <module>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/angr/knowledge_plugins/variables/variable_manager.py", line 16, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 857, in get_code
  File "<frozen importlib._bootstrap_external>", line 525, in _compile_bytecode
MemoryError
